modules/blocks/verify.js

```diff
@@ -45,14 +45,15 @@ __private.lastNBlockIds = [];
  * @todo Add description for the class
  */
 class Verify {
-	constructor(logger, block, transaction, db) {
+	constructor(logger, block, transaction, db, config) {
 		library = {
 			logger,
 			db,
 			logic: {
 				block,
 				transaction,
 			},
+			config,
```

Comment:
```
config: {
	loading: {
		snapshotRound: config.loading.snapshotRound,
	},
},
```

Reply:
@4miners with this usage we will override the config object and loose the other properties?

Reply:
Yep